src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -117,9 +118,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Salary updatedSalary = editPersonDescriptor.getSalary().orElse(personToEdit.getSalary());
         Info updatedInfo = editPersonDescriptor.getInfo().orElse(personToEdit.getInfo());
         PrevDateMet updatedPrevDateMet = editPersonDescriptor.getPrevDateMet().orElse(personToEdit.getPrevDateMet());
+        ScheduledMeeting scheduledMeeting = personToEdit.getScheduledMeeting();
 
         return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedFlag,
-                updatedTags, updatedPrevDateMet, updatedSalary, updatedInfo);
+                updatedTags, updatedPrevDateMet, updatedSalary, updatedInfo, scheduledMeeting);
```

Comment:
It would be better if `updateScheduledMeeting` is used. To make them look the same.